resize.php

```diff
@@ -0,0 +1,71 @@
+<?php
+require(dirname(__FILE__).'/config/config.inc.php');
+$url = filter_input(INPUT_GET, 'url');
+
+preg_match('/^(img\/([^\/]*)\/(?:\d\/)*)(\d+)-([_a-z0-9]+)\.([a-z]+)$/', $url, $matches);
```

Comment:
We must harden this script by also handling bad URLs. If URL does not match expected format, `$matches` is an empty array.